src/systems/ackermann_steering/AckermannSteering.cc

```diff
@@ -729,14 +729,23 @@ void AckermannSteeringPrivate::UpdateVelocity(
   // Convert the target velocities to joint velocities and angles
   double turningRadius = linVel / angVel;
   double minimumTurningRadius = this->wheelBase / sin(this->steeringLimit);
-  if ((turningRadius >= 0.0) && (turningRadius < minimumTurningRadius))
+  if (fabs(linVel) > 0.0)
   {
-    turningRadius = minimumTurningRadius;
+    if ((turningRadius >= 0.0) && (turningRadius < minimumTurningRadius))
+    {
+      turningRadius = minimumTurningRadius;
+    }
+    if ((turningRadius <= 0.0) && (turningRadius > -minimumTurningRadius))
+    {
+      turningRadius = -minimumTurningRadius;
+    }
   }
-  if ((turningRadius <= 0.0) && (turningRadius > -minimumTurningRadius))
+  // special case for angVel not zero and linVel zero
+  else if (fabs(angVel) >= 0.001)
   {
-    turningRadius = -minimumTurningRadius;
+    turningRadius = (angVel/fabs(angVel))*minimumTurningRadius*1.5;
```

Comment:
The `1.5` multiplication appears to be unneeded (everything is working as expected when removed). If it is needed, can you explain why?
```suggestion
    turningRadius = (angVel / fabs(angVel)) * minimumTurningRadius;
```

Reply:
Well, it comes down to the calculation of the turningRadius. This "1.5 x increase in size" of the turning radius only applies when there is zero linVel. For some reason it felt odd to force the wheels completely to their limits even if the angVel was only .001 but the linVel is zero simply because it's not being commanded, seems to have the effect of making the vehicle "dance" as that angle can be very aggressive. From a strictly "this should only be used with teleop or twist" perspective I see your point though.

I suppose maybe it's my overall dislike of the Ackermann Plugin using the twist message for angular velocity instead of setting a steering angle slowly working it's way through into the code. 

But for sake of cohesiveness of the intention of the plugin, I'll commit that suggested change.
Hopefully we can get this fix in ASAP and rolled into a spot release.